Dockerfile

```diff
@@ -2,21 +2,9 @@
 # To build, run docker build --rm --tag=hmda/grasshopper-parser .
 # A container can be started by running docker run -ti -p 5000:5000 hmda/grasshopper-parser
 
-FROM ubuntu:latest
+FROM python:2.7.9-onbuild
 MAINTAINER Juan Marin Otero <juan.marin.otero@gmail.com>
```

Comment:
I was actually thinking about what/who we should put in the `MAINTAINER` field for all these projects.  I _think_ we want it to be the same across all of our images...and it sounds like it makes better use Docker's cache if its the same too:

* http://crosbymichael.com/dockerfile-best-practices.html

Perhaps a hmdaops-docker@cfpb.gov address at some point?

For now, I'll switch if over to me. :)